### PR TITLE
Pulling prone mobs slows you down, fireman carrying

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Pulling a prone mob slows you down.
This should be anything that makes spessman horizontal. Currently it is resting, crit, knocked down, unconscious, or dead. tell me if I missed any.
Having them buckled to anything does not slow you down.
You also now can fireman carry prone humans as a human. Aggressive grab then click and drag onto you, and you will begin fireman carrying. This requires a free hand. This causes a slight slowdown, but not nearly as much as dragging does.

## Why It's Good For The Game

Prevents the classic "stun and beat the shit out of while zipping off" which is obnoxious and I don't think anybody likes besides the people who do it.
Makes it so if you feel like being a cunt you can rest while being arrested to make you a pain to take back.
Makes it harder to steal bodies and move away before anybody can properly react, but makes dragging bodies riskier at the same time
Fireman carrying adds an element of risk reward, it takes a little while of standing still to pick them up and also slows you down slightly. More useful to move someone a long distance out of a relatively safe area than a quick pull away.

## Changelog
:cl:
balance: Dragging a prone (knocked down, resting, in critical) mob slows you down.
add: You can now fireman carry. Fireman carrying can be done by humanoids, on prone humanoids. Aggressive grab then click and drag them to you to begin fireman carrying a prone humanoid. It requires a free hand.
balance: Carrying another human applies a slight slowdown.
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
what about silicons and simplemobs

**Comment by actioninja** [comment]
> what about silicons and simplemobs

What do you mean? If the simplemob can rest, yeah, they can rest to be slowed down. I don't believe silicons can rest, and they are affected by the drag slowdown.

**Comment by ExcessiveUseOfCobblestone** [comment]
is there not a dead check or is that intentional?

**Comment by actioninja** [comment]
I assumed that a critical check would cover dead as well, is that wrong?

**Comment by ExcessiveUseOfCobblestone** [comment]
I can only find

https://github.com/tgstation/tgstation/blob/6c2f37c6b1f56f7432dce5cf78bc0aed6845765e/code/modules/mob/living/living.dm#L371-L372

**Comment by Suicidalpickles** [comment]
will this slow down aggrograbs on bodies down even more, or do they retain the same pull speed since they're already pretty slow?

**Comment by imsxz** [comment]
these changes hurt me physically but i know the game needs it to have combat that isnt aids godspeed

**Comment by mvm900** [comment]
good idea and it makes the bad users angry

**Comment by 4dplanner** [comment]
Making it more annoying to help people (such as drag to medbay) will lead to less people helping, which will be irritating. The arrest part will probably also get old fast as sec.

**Comment by 4dplanner** [comment]
Also dragging off a body while hitting it is good in that it lets you do SOMETHING against a stronger force, rather than being forced into 100% running because of how easy patching someone up from hit and run attacks is in this game.

**Comment by Therapist1** [comment]
This will make rollerbeds actually have a legitimate use besides rocketing down the halls. As for arrests, just give them extra time for resisting. Good change all and all.

**Comment by ExcessiveUseOfCobblestone** [comment]
just use BSbodybags to detain perps lol

**Comment by Mickyan** [comment]
you can't make killing harder because healing is too easy

you can't make healing harder because killing is too easy

🤔

**Comment by anconfuzedrock** [comment]
> Making it more annoying to help people (such as drag to medbay) will lead to less people helping, which will be irritating. The arrest part will probably also get old fast as sec.

99.9% of the time someone drags someone to medbay, they drag out 35% of their fucking blood, this will at least slow them down so they can be called a dumbass, or maybe even make them use a locker! 
Speed drugs still exist if you need to drag people to hell.

**Comment by imsxz** [comment]
does it slow down silicons/simple mobs that drag people? seems like a big holoparasite buff since the main counter was the drag'n'beat

**Comment by ExcessiveUseOfCobblestone** [comment]
just do it on a rolly chair lol

pocket metal will be the new meta

**Comment by anconfuzedrock** [comment]
#bodybagmasterrace

**Comment by wesoda25** [comment]
Put more thought into a better solution, in cult/rev/tiding situations no one has time to stun and cuff, you usually need to stun them until you reach a safe area. This isn’t a traitor nerf, its a security one.

**Comment by Shadowflame909** [comment]
Isn't it a cult nerf though @wesoda25 

FU-MAJIN!

>Goes and drags the Hop away really slowly

**Comment by ATH1909** [comment]
This is a big holoparasite nerf; one of the best things about having a holoparasite is that they can pull you away from sec officers while you're stunned so that you won't be cuffed/you can escape. Now, if a holoparasite tries that, they'll just get shot by lethals and die because the sec officers will easily be able to keep up with them.

**Comment by PKPenguin321** [comment]
Would be cool if there was a way to fireman carry a downed person. Maybe grab intent -> Aggro grab -> Clickdrag to self? There would obviously be a delay while you're lifting them up and it could use a hand slot

**Comment by Shadowflame909** [comment]
@PKPenguin321 That'd be a nice alternative.


This mass rebalances PR just doesn't cut it chief.


In my opinion, there are literally too many game modes and things in this game that function around grabbing your partner and being able to zip away.


Huge Nerf. It may not be a feature that is wanted, but it's surely one that needs to be there. You literally have to replace it with something

**Comment by wesoda25** [comment]
Tbh it seems as if you’re in this fervor of game changing PRs and haven’t stopped to consider whether or not this will actually improve the game. 

Dragging at full speed isn’t really a great mechanic, and it makes for shit gameplay like stun, grab, and run. However, its not necessarily a bad mechanic, and adding a slowdown would result in many good uses being nerfed. Tasks normal to accomplish usually will become challenging with this, and make certain aspects of gameplay unenjoyable. 

Now that it has been called to attention, perhaps passive drag does need changing. However, I think it requires something a bit more articulate than this, seems sort of like a rushed PR, the first solution that came to mind.

**Comment by Shadowflame909** [comment]
Alternative Alternative Idea.


What if you just increased the amount of damage dragging does to an injured person. Sounds like an easy slight change that doesn't cause a fester of salt and hate to form.


I'm pretty sure dragging people worsens blood loss, so if you do that + the new Epi-Pen and oxygen healing nerf PR. It'll increase the use of a medical doctor more as they need to help heal blood loss.


Bam. A slight change that'll improve departmental interaction

**Comment by actioninja** [comment]
@Shadowflame909 
Massively increasing damage from dragging just means people would powergame it. Stab them, then stun them, and run around with them for a bit. Fast and easy kills with only a stunprod and something sharp.
I also don't really think it _needs_ to be there. It affects everyone, revs can't easily save eachother or heads from being stabbed but at the same time, heads or sec can't easily drag away a fucked up head from other revs to get backup.
I also don't like your rock paper scissors comment because nerfs are sometimes necessary when the standard of power is astronomical and raising everything to meet it would just be insanity. Hardstuns for example, are essentially an win. Unless they have adrenals, if you've stun someone it's over and you've already won. There's no room for tweaking besides the length of the stun, and if it's longer than it takes to cuff someone it might as well be infinite. Adding counters to a hardstun is literally "makes you not hardstunned anymore" or "kills them extremely fast before you can get hardstunned" which are both their own issues and complications that don't need to be there if you just nerf or remove hardstuns.


@wesoda25 This wasn't even my idea, this is something that's been talked about for months and months with nobody ever actually doing it. I thought it was a good idea so I implemented it. I have thought about potential ramifications, and I don't think any of them would be particularly severe. Can you give an example of something that would be dramatically harder? All of the situations I can think of that would be much, much harder are really obnoxious shit nobody likes, most other situations have ways to mitigate it at the very least.

> Would be cool if there was a way to fireman carry a downed person. Maybe grab intent -> Aggro grab -> Clickdrag to self? There would obviously be a delay while you're lifting them up and it could use a hand slot

Solid idea actually. I'll see if I can implement this. Makes an neat trade off of dragging vs carrying, one being able to be done quickly but slows you down, the other requiring some times standing in place but doesn't injure the person and also you can move a bit quicker.

**Comment by Shadowflame909** [comment]
@actioninja  You see MR Ninja. I'm not disagreeing that it affects everyone. But I am disagreeing on the amount that it does affect everyone.

You have Cults, Xenos, Revs, Security Teams, Medical Doctors, Holoparasites, Nukies who need to carry the captain who embedded the Nuke Disk inside of him, and Slaughter Demons, Morphs occasionally, whatever Mobs thrive off of hit and run mechanics. For different reasons maybe, but they all use them.

You're telling me that because you don't like it when Revs go "zoom zoom" all of a sudden the mechanics of all these other things need to get fucked over because they, in fact, require that "zoom-zoom".

Well, that's absurd, and I would better hope you'd agree on an alternative because like it has been said here. You're literally just applying one large bandaid, over separate and different issues.


If this is a revs problem and not a problem of Hit and Run Simple Mobs. Then I would hope you'd make a change that'd separate different antags who literally need to use this mechanic to accomplish their objectives.


Lastly, on the matter of my Rock Paper Scissors argument. I think it applies here greatly. You say that"Hardstuns for example, are essentially an win. Unless they have adrenals, if you've stun someone it's over and you've already won. "

Well, isn't that still a counter my dear friend?

In a situation like this, you have a mechanic that works fine. But it works too well, and we see it play out too commonly. I say this mechanic works like this because it's wholly necessary for the function of several antagonists in this game. But you say that since this is nerfing everybody who can use it, that this PR is also fine. 

You're beating the rock in rock versus scissors here, by applying a bandaid fix that doesn't properly address everything that needs to be addressed. Because several antagonists rely on this mechanic. 

TLDR: Fix no good!1! Bandaid does not address the issue properly!!! Please try something else!1!

**Comment by Fox-McCloud** [comment]
Giving all the validhunting mobs even more chances to beat you to a bloody pulp.

**Comment by Suicidalpickles** [comment]
I can see this being more so a hassle on cult than revs, since you'd need to spam more than one stunhand down onto the same person you're kidnapping, which in turn would pile more damage onto you. 

Pretty much might as well start using hallway runes more or killing even more folk instead of bothering to haul to convert ever.

**Comment by Militaires** [comment]
make sure the slowdown from aggressive grabs doesnt stack on top of this

**Comment by dr-bee** [comment]
> 
> 
> Giving all the validhunting mobs even more chances to beat you to a bloody pulp.

You say this like it is a bad thing. Fearing a mob of angry crewmen killing you for openly murdering people should be a legitimate fear of antagonists. The fact that it often isnt a legit fear for antags is a testament to how much antag power has creeped over the years.

**Comment by Shadowflame909** [comment]
@dr-bee 

I wish that argument was made when actionninja made disarms useless

**Comment by dr-bee** [comment]
> 
> 
> @dr-bee
> 
> I wish that argument was made when actionninja made disarms useless

He didnt make them useless he made them consistent, Try thinking where you shove someone and when instead of shoving and wondering where the RNG free win went.  Also that is not what this PR is about.

**Comment by Fox-McCloud** [comment]
>The fact that it often isnt a legit fear for antags is a testament to how much antag power has creeped over the years.

The more egalitarian you make the crew vs antag dynamic, the more people will be willing to validhunt because it means they have a real chance at winning the engagement. 

The counter-argument to this is "but if antags are strong then they'll just kill everyone". Yes, if security doesn't work together or multiple crewmembers don't get involved.

Having literally anyone on the station be a threat to your plans can and will actively encourage regular crewmewmbers to get involved in trivial scuffles, because they can win and they get to be the hero in the process.

**Comment by dr-bee** [comment]
> 
> 
> > The fact that it often isnt a legit fear for antags is a testament to how much antag power has creeped over the years.
> 
> The more egalitarian you make the crew vs antag dynamic, the more people will be willing to validhunt because it means they have a real chance at winning the engagement.
> 
> The counter-argument to this is "but if antags are strong then they'll just kill everyone". Yes, if security doesn't work together or multiple crewmembers don't get involved.
> 
> Having literally anyone on the station be a threat to your plans can and will actively encourage regular crewmewmbers to get involved in trivial scuffles, because they can win and they get to be the hero in the process.

We arent talking about one crewmember, we are talking about a mob of them. Antags will have 1v1 superiority due to their gear even after this change and the addition of shoving. The only difference is that they will have to actually pick their battles more instead of being able to fight entire groups of people by yakkety saxxing around smacking them all.

This change does not only effect antagonists as well. Think in terms of nuke ops. Dragging away wounded crew-members from a firefight will now leave that person open to being easily shot. This change mostly just makes picking the location of your fight more important than it was, as you cant relocate your target or save your allies as easier as before, which is honestly a good thing.

Requiring more thought from players is not a bad thing.

**Comment by Shadowflame909** [comment]
Making people carry metal in their backpacks to bypass the slowdown sounds like a timegate for no reason. Unless this is the thinking we should do

**Comment by actioninja** [comment]
> Making people carry metal in their backpacks to bypass the slowdown sounds like a timegate for no reason. Unless this is the thinking we should do

I'm working on a fireman carry system that would basically just be this but less cheesy. Timegates can be a good thing, they force you to stand in the same place doing nothing else for a sustained period which can be a really bad idea depending on the situation.
You could also fold up a rollerbed and carry it with you if you're a medic or security dealing with a lot of greyshits.

**Comment by Shadowflame909** [comment]
@actioninja You don't have to do much then. Considering we have an alternative carry in the game already that IRL couple players use.

**Comment by Mickyan** [comment]
Security protip: you can shake people up from resting so if someone tries it just keep clicking on them with an empty hand for the 10 seconds it takes to get to security and then feel free to throw them in the gulag for resisting arrest

**Comment by ExcessiveUseOfCobblestone** [comment]
"timegate" on something that's provided en masse roundstart???

You could prob just throw people across halls for faster travel anyways lol.

**Comment by anconfuzedrock** [comment]
We got piggybacking, just need it to be less of a team effort. People on backs can’t act regardless.

**Comment by actioninja** [comment]
> We got piggybacking, just need it to be less of a team effort. People on backs can’t act regardless.

Piggybacking was actually so hardcoded I had to refactor a good amount of the human buckling system to implement fireman carrying. And I'm still working on it, for that matter.

**Comment by anconfuzedrock** [comment]
welp so much for my plans to allow you to buckle yourself to the floor using magboots

**Comment by ExcessiveUseOfCobblestone** [comment]
> > We got piggybacking, just need it to be less of a team effort. People on backs can’t act regardless.
> 
> Piggybacking was actually so hardcoded I had to refactor a good amount of the human buckling system to implement fireman carrying. And I'm still working on it, for that matter.

Kevinz-code

**Comment by anconfuzedrock** [comment]
> You have Cults, Xenos, Revs, Security Teams, Medical Doctors, Holoparasites, Nukies who need to carry the captain who embedded the Nuke Disk inside of him, and Slaughter Demons, Morphs occasionally, whatever Mobs thrive off of hit and run mechanics. For different reasons maybe, but they all use them.


All those people (except sec officers, who can negate it by cuffing and shaking up) deserve this nerf, really- even for mds, it’s better for the patient’s health if docs have to get rollerbeds and slow down their “dragging all his blood out” that happens way too often.

>simple mobs xenos and laughter demons gain the ability to fireman’s carry when



They already did, called bloodcrawl and vore respectively. If this actually effects simplemobs, whoop de doo, even morphs can just eat people on the spot.
Also personally offended that slaughter demons arr completely forgotten compared to laughter demons

**Comment by Shadowflame909** [comment]
N0 way Mr confused rock.

>Xenos

This is bad because Xeno Vore is a double-edged knife. It literally gibs people and is a good way to kill Lings. It's useful for vent-crawling sure, but sometimes If my targets near death or pregnant. I'd rather not take the risk. Fat Nerf!

>Laughter/Slaughter-Demons

How are they supposed to track blood if their entire gimmick is moving really fast once out of blood, and then rapidly slow down? Their dragging blood technique is really hitting a flat nerf. Which is why this gets disapproved.

>Cult/Revs

Eh. Fireman carry. More worried about the cult, but I guess I should just pro-actively remove headsets. So I'm not too worried.

>Holoparasites

Dude, This is not needed for them. You think they're op, but you know what else you can buy that's way better? Stealth-Box, Emags, Real Adrenals, Stun immunity Kung Fu, Dual E-bows.  Usually your clunky partner will get in the way, as they're fodder for bullets and guns because they're already slower then you.

If you drag off a dying partner, you have little hope of healing them to life as you'll be valided by everything, and shot at by everything whilst running and looking for a medibot.


Besides, a Holoparasite is as good as a Cuff-breaker implant. Once exposed that you have it, you're really cheesed if you get seen again.

**Comment by anconfuzedrock** [comment]
You are wrong.

**Comment by Shadowflame909** [comment]
I disagree with your statement

**Comment by Ragicil** [comment]
you can already crawl and disarm after being slipped, what is it with tg trying to turn the entire game into a pacifist hugbox. just remove combat altogether already so we can avoid the next few years of constant passive aggressive prs

**Comment by actioninja** [comment]
> you can already crawl and disarm after being slipped, what is it with tg trying to turn the entire game into a pacifist hugbox. just remove combat altogether already so we can avoid the next few years of constant passive aggressive prs

Hi, reddit!

**Comment by Shadowflame909** [comment]
anyone who makes a string of nerf PRs is secretly 4dplanner and oranges and whoever else


im onto you actioninja

**Comment by Anonymous-Player** [comment]
This is a good idea.

**Comment by Anonymous-Player** [comment]
@Shadowflame909 
Have you ever considered that Xenos and Slaughter Demons are much stronger than humans and can drag humans at now normal speed? This is just for humans.

**Comment by Shadowflame909** [comment]
To be quite honest, I'm simply baffled at the argument being presented to me.

>A wizard spawn and a round ender are strong. So this nerf is good


Well the nuclear device is strong too. So why don't you nerf that down to human speed

**Comment by Anonymous-Player** [comment]
What is so confusing? There is nothing mandating that Holoparasites, Xenos, or Slaughter Demons have to be effected by this change. You can just create a separate PR that returns their speed when pulling to be what it was pre-nerf. 

The main reason this is being changed is not for the edge cases of Slaughter Demons and Xenos, but the majority case of humans. Why do you people ALWAYS argue against changes based on edge cases? Edge cases make up 1% of game time and can be accounted for in a separate PR.

**Comment by ExcessiveUseOfCobblestone** [comment]
> you can already crawl and disarm after being slipped, what is it with tg trying to turn the entire game into a pacifist hugbox. just remove combat altogether already so we can avoid the next few years of constant passive aggressive prs

image

**Comment by Shadowflame909** [comment]
If the PR owner isn't addressing these "edge cases" chances are they're intentional.

**Comment by ExcessiveUseOfCobblestone** [comment]
> If the PR owner isn't addressing these "edge cases" chances are they're intentional.

extremely bold assumption

**Comment by Shadowflame909** [comment]
Action ninja could respond to his thoughts on this effecting non humans then.

**Comment by actioninja** [comment]
> If the PR owner isn't addressing these "edge cases" chances are they're intentional.

Mostly true

> extremely bold assumption

Also mostly true

**Comment by Shadowflame909** [comment]
What did he mean by this



I hope we find out



I am against it though

**Comment by anconfuzedrock** [comment]
I had a nightmare last night that shadowflame forced cultists to be immune to drag slowdowns

**Comment by Poojawa** [comment]
I do appreciate Stand yeets being nerfed with this. Good work.

**Comment by Shadowflame909** [comment]
I just think that the lack of mention, and basically ignoring how this effects non-humans and other mobs isn't right.


#Slaughterdemonlivesmatter


Also no Holoparasites are weak shit and the only way to die to one is to be caught off guard, or for a bug to be abused. Looking at you Explosive Holoparasites

**Comment by actioninja** [comment]
> I had a nightmare last night that shadowflame forced cultists to be immune to drag slowdowns

I'm still laughing about this

**Comment by anconfuzedrock** [comment]
I couldn’t make it up if I tried boss

**Comment by actioninja** [comment]
> I just think that the lack of mention, and basically ignoring how this effects non-humans and other mobs isn't right.
> 
> #Slaughterdemonlivesmatter
> 
> Also no Holoparasites are weak shit and the only way to die to one is to be caught off guard, or for a bug to be abused. Looking at you Explosive Holoparasites
> 
> Edit: No seriously, if he's really going to add fireman carrying. I give no hecks about the effect it has on Human Versus Human Combat.
> 
> But simple mob antagonists really rely on dragging. A whole lot of them. This PR needs to address that because no codebase likes half-hearted solutions and preventable bugs.

How about this: I'll add a var to mobs that can be toggled to disable any drag slowdown. If this ends up being a catastrophic nerf to mobs like the slaughter demon and benos you can make a 1 line PR to make them not affected by this.

**Comment by Shadowflame909** [comment]
Yeah ok. Since I'm the only one arguing this,  that's a satisfying conclusion

**Comment by ExcessiveUseOfCobblestone** [comment]
in other words he no longer has to put up with you but we still do when the inevitable following PRs come out.

**Comment by Shadowflame909** [comment]
Well, this is now a huge simple mob Nerf. So it'll be interesting to see how many simple mobs receive one line changes, and which ones stick with it.

**Comment by PKPenguin321** [comment]
Only complaint is that I expected fireman carry to not give slowdown, with the trade-off being how it takes a delay and a hand slot to do. Like the lack of slowdown was the whole point

**Comment by Poojawa** [comment]
it's just piggyback riding but with someone being incapped.

**Comment by actioninja** [comment]
@PKPenguin321 it's a very slight slowdown, and the idea is to actually give a point to things like the rollerbed. If you want to move people around at full speed without fucking them up if they're in crit, you need to plan ahead.

**Comment by PKPenguin321** [comment]
Guess we'll see

**Comment by Shadowflame909** [comment]
The rollerbed already has a point though. It's so that you can take an injured person to medbay without them losing blood.

I would hope you'd retract from giving the firemen carry a speed nerf as well. If you make it so, "This item does this action better then without having this item" You're going to see that item not being used for its original use at all.


Or to put it more simply, assistants and antagonists are going to steal Rollerbeds from actual medical personnel. Making them even more useless.

Since the Rollerbed already has a use, if you want them to be more useful. Just aggravate that use in a separate PR. Like making blood-loss from being dragged around on the floor more severe.

**Comment by ExcessiveUseOfCobblestone** [comment]
Can you craft rollerbeds...?

**Comment by Shadowflame909** [comment]
Nope.

But the wiki lead me to a funny section where it talks about how beds were once used for ERP on /tg/station.

Classic.

**Comment by ExcessiveUseOfCobblestone** [comment]
They should be craftable at the med techfab at the cost of lockers giving the same slowdown as fireman carry ;)

**Comment by Mickyan** [comment]
I think pacifists are going to be unable to fireman carry since they can't aggressive grab

Which I think might be a bug/oversight? This seems to imply it should stop you from throwing mobs which already needs an aggressive grab
https://github.com/tgstation/tgstation/blob/d5b261572fb0a1c6c4ef41861694d60dbf7817ac/code/modules/mob/living/carbon/carbon.dm#L160-L166

**Comment by PKPenguin321** [comment]
Also just remembered that aggro grab -> clickdrag to self is shared by swallowing mobs whole (which you can do to monkeys if you're obese, or if you're a xenomorph)

**Comment by Shadowflame909** [comment]
Yeah, this PR is quite the allergic reaction to a problem that doesn't exist.

**Comment by anconfuzedrock** [comment]
the carry is the allergic reaction to the problem that doesn't exist
very excited for a nerf to the cultists that can carry four stunmutes in their arms at once and for the nerf to the unstunnable demons who cannot be caught unless they willingly come out and are too dumb to escape

**Comment by Shadowflame909** [comment]
>Because I cant stun a simple mob that only appears on wizard rounds, it must need a nerf

>Because cult is a shit mode and they get invisible instant stuns and can make four of them. We must nerf everything else

Holy shit dude. This is literally like going Tsunamis are the best solution to global warming.

You're going too far off the deep end. If you have a problem with those individual features, Mass Nerfs that either makes a feature redundant or breaks it entirely isn't the solution.

I also disagree with you on Slaughter Demons. You're angry that simple mobs aren't stunnable? That's literally every simple mob ever. They offer really good Loot too, And they already slow down super fast to below human speed. So you can easily outrun them as long as you're not near any blood. 

It's a problem that doesn't exist.

Also cobby, I feel like them being craftable would overcomplicate things. In essence, it'd be like, "You can't go from Point A to Point B without this item. So every round you must make sure to head to medbay and give up some inventory space to store this item in your bag in the first place." When you should just be able to use the alternative Fireman carry feature in the first place.

The fireman carry was a fake solution to throw heat off his back. This PR is still the same as it was originally. Just with a Poster covering the gaping hole in the wall.

**Comment by ExcessiveUseOfCobblestone** [comment]
> Also cobby, I feel like them being craftable would overcomplicate things. In essence, it'd be like, "You can't go from Point A to Point B without this item. So every round you must make sure to head to medbay and give up some inventory space to store this item in your bag in the first place." When you should just be able to use the alternative Fireman carry feature in the first place.

It's not mandatory unless you're carrying people constantly (security maybe?). It also isn't inventory space unless you have a BSB maybe, it's a handslot OR the "pulling" slot. If you need one that badly then using up that slot shouldn't be an issue, and it becomes a very balanced item.

This isn't including the fact that, unlike lockers, the only thing that can go on the bed are mobs. You could say that beds move faster because it's easily visible that you're dragging someone away, unlike a locker/crate where you could be dragging ore (rip) or parts.

That's not to suggest I don't want the fireman carry, I just think there's merits in the PR even without it.

**Comment by Shadowflame909** [comment]
My point on the fireman carry was just that. It's not the all for one alternative to pulling mobs. Since you have to steal stuff from medbay if you want to do it very efficiently. Not Epic.

This PR needed forethought. It contained lies and silence.

**Comment by actioninja** [comment]
> Edit: Actioninja why do you do this to the player base, First it was pretending to re-add pre-scanning. Then it was pretending Disarms were still useful. And now it's pretending like you've added an alternative to your nerf. It's one big sham, fraud, and since such an event has become so common with your Pull Requests. I think I shall see past them for any future ones you do.

lol

**Comment by anconfuzedrock** [comment]
I understand you have goldfish memory shadowflame,  but I thought you’d remember that I am pleased with everything this nerfs, not just memons and cucktists.

**Comment by Shadowflame909** [comment]
Oh yeah, what if I make a Sentient Pet Rock Simple Mob.

Too bad it can't drag you to medbay efficiently before you bleed out

Because its dragging speed is terrible.


But this unironically breaks slaughter-demons.

**Comment by anconfuzedrock** [comment]
The joke is, dear retard, that a slower dragging speed would stop people bleeding out, because people bleed out extremely slowly unless they're currently being fucking moved. This doesn't break slaughter demons. the broken part of slaughter demons is they're physically impossible to stop unless they're too aggressive and fuck up. I personally don't hate the idea that what isn't even the main round antag can be evaded at least partially by sticking together.

**Comment by PKPenguin321** [comment]
Don't say the r word you fool

**Comment by anconfuzedrock** [comment]
I have the R word pass

**Comment by plapatin** [comment]
why are you like this

**Comment by anconfuzedrock** [comment]
I have the r word pass, because I’m a retard.

**Comment by ExcessiveUseOfCobblestone** [comment]
> I think pacifists are going to be unable to fireman carry since they can't aggressive grab
> 
> Which I think might be a bug/oversight? This seems to imply it should stop you from throwing mobs which already needs an aggressive grab
> [tgstation/code/modules/mob/living/carbon/carbon.dm](https://github.com/tgstation/tgstation/blob/d5b261572fb0a1c6c4ef41861694d60dbf7817ac/code/modules/mob/living/carbon/carbon.dm#L160-L166)
> 
> Lines 160 to 166 in [d5b2615](/tgstation/tgstation/commit/d5b261572fb0a1c6c4ef41861694d60dbf7817ac)
> 
>  if(pulling && isliving(pulling) && grab_state >= GRAB_AGGRESSIVE) 
>  	var/mob/living/throwable_mob = pulling 
>  	if(!throwable_mob.buckled) 
>  		thrown_thing = throwable_mob 
>  		stop_pulling() 
>  		if(has_trait(TRAIT_PACIFISM)) 
>  			to_chat(src, "<span class='notice'>You gently let go of [throwable_mob].</span>")

@actioninja tested?

**Comment by actioninja** [comment]
@ExcessiveUseOfCobblestone I finally actually tested this, and yes, pacifists cannot fireman carry.
Two options of how I can fix this:
Pacifists can fireman from a passive grab
Pacifists can aggressive grab but can't do any aggressive actions.

Which would be be better?

**Comment by ExcessiveUseOfCobblestone** [comment]
The latter I think unless you have a good reason to do the former.

**Comment by optimumtact** [comment]
Thog say, for now, want to see how meta shakes out, but thog interested in seeing this later

**Comment by Ratvar** [comment]
thog, look at dis again

**Comment by Shadowflame909** [comment]
If I had a nickel for every time I necro'd a forgotten nerf.

I'd be broke.

**Comment by optimumtact** [comment]
@Ratvar it is on my list, don't worry, I do intend to remove the stun + click drag into maint to kill at leisure mechanics
